packages/db-manager/README.md

```diff
@@ -59,6 +59,7 @@ npm install --save @middy/db-manager
 - `config`: configuration object passed as is to client (knex.js by default), for more details check [knex documentation](http://knexjs.org/#Installation-client)
 - `client` (optional): client that you want to use when connecting to database of your choice. By default knex.js is used but as long as your client is run as `client(config)` or you create wrapper to conform, you can use other tools. Due to node6 support in middy, knex is capped at version `0.17.3`. If you wish to use newer features, provide your own knex client here.
 - `secretsPath` (optional): if for any reason you want to pass credentials using context, pass path to secrets laying in context object  - good example is combining this middleware with [ssm](#ssm)
+- `secretsParam` (optional): override the connection parameter when setting the password directly from ssm using `secretsPath` or with `rdsSigner`. This is ignored when passing an object in. Default: `password`.
```

Comment:
It can be confusing to user when should he provide what - maybe it would be worth bo add example here that showcases both? Other than that I will approve it on my side

Reply:
I was going to, but I couldn't find one. Secret Manager doesn't need it, and the popular db clients I looked at all use `password` as their parameter. It's more of an option for the super edge case we don't know about yet.